gonanoid.go

```diff
@@ -44,6 +45,13 @@ func getMask(alphabet string, masks []uint) int {
 
 // Generate is a low-level function to change alphabet and ID size.
 func Generate(alphabet string, size int) (string, error) {
+	if len(alphabet) == 0 {
```

Comment:
I think we can check here that the length of alphabet is less than 256. Longer alphabet wouldn't work as we use randomly generated bytes for selecting random letter from the alphabet, thus the 256 characters limit.